crates/nu-utils/standard_library/tests.nu

```diff
@@ -63,8 +63,76 @@ def test_path_add [] {
     }
 }
 
+
+def test_dirs [] {
+
+    def "myassert eq" [
+        left:any 
+        right:any 
+        msg?:string = "..."
+        --verbose:bool = false (-v)  # enable to see successful tests and values of unequal tests
+    ] {
+        if $left != $right {
+            let start_span = (metadata $left).span.start
+            let end_span = (metadata $right).span.end
+            let fail_msg = (if $verbose {
+                                $"\n    left: ($left|debug)\nnot equal to\n   right: ($right|debug)"
+                            } else {""})
+            error make {msg: $"Assertion failed checking ($msg)($fail_msg)", label: {text: "Values not equal" start: $start_span end: $end_span}}
+        } else {
+            if $verbose {
+                echo $"check succeeded: ($msg)"
+            }
+        }
+    }
+    
+    # need some directories to play with
+    let base_path = ($"tmp_(random uuid)" | path expand )
+
+    try {
+        mkdir $base_path ($base_path | path join "a") ($base_path | path join "b")
```

Comment:
maybe `($base_path | path join "a")` and `($base_path | path join "b")` could be refactor into two variables, `a` and `b`, for easier readability? :yum:

Reply:
OK